### PR TITLE
Update the default configuration of ECAL modules

### PR DESCRIPTION
**Pull request by fwyzard**
Update the default configuration of `EcalRawToDigiGPU` and `EcalUncalibRecHitProducerGPU`:
  - enable the ECAL pulse timing computation in the offline workflow;
  - change the ECAL GPU digis label to match the CPU ones.

Other clean up:
  - remove an unused cfi file;
  - rename `EcalUncalibRecHitMultiFitAlgo_gpu_new` to `EcalUncalibRecHitMultiFitAlgoGPU`.

### CODE REVIEW

**Comment by fwyzard** [comment]
@thomreis could you double check these changes ?

**Comment by fwyzard** [comment]
:construction: Validation running at fu-c2a02-35-03:/data/user/fwyzard/patatrack/validation/run_598.iB50AON7te ...

**Comment by fwyzard** [comment]
Standard workflows seem to be running fine.
The HLT customisation also runs fine (tested privately)

File: RecoLocalCalo/EcalRecProducers/python/ecalMultiFitUncalibRecHit_cff.py [hunk 1]
```diff
@@ -19,8 +19,8 @@
 # ECAL multifit running on GPU
 from RecoLocalCalo.EcalRecProducers.ecalUncalibRecHitProducerGPU_cfi import ecalUncalibRecHitProducerGPU as _ecalUncalibRecHitProducerGPU
 ecalMultiFitUncalibRecHitGPU = _ecalUncalibRecHitProducerGPU.clone(
-  digisLabelEB = cms.InputTag('ecalDigisGPU', 'ebDigisGPU'),
-  digisLabelEE = cms.InputTag('ecalDigisGPU', 'eeDigisGPU'),
+  digisLabelEB = cms.InputTag('ecalDigisGPU', 'ebDigis'),
```
Comment (thomreis) [inline RecoLocalCalo/EcalRecProducers/python/ecalMultiFitUncalibRecHit_cff.py:22]: Remove `cms.InputTag` as well here as done in `HLTrigger/Configuration/python/customizeHLTforPatatrack.py`?

Comment (thomreis) [reply RecoLocalCalo/EcalRecProducers/python/ecalMultiFitUncalibRecHit_cff.py:22]: Ah well it is already merged. Did not see that.

Comment (fwyzard) [reply RecoLocalCalo/EcalRecProducers/python/ecalMultiFitUncalibRecHit_cff.py:22]: thanks for the comment, I can add this later anyway

**Pull request merged**